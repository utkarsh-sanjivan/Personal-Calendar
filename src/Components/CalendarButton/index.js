import React from 'react';
import { Button } from 'antd';
import './style.css';

const CalendarButton = props => (
    <Button 
        {...props}
    />
)

export default CalendarButton;
