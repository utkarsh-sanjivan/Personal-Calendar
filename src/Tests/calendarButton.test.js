import React from 'react';
import { create } from "react-test-renderer";
import CalendarButton from './../Components/CalendarButton/index';

test('Test CalendarButton Component', () => {
const component = create(<CalendarButton 
    className=''
    onClick={() => {}}
  />);
  expect(component.toJSON()).toMatchSnapshot()
});