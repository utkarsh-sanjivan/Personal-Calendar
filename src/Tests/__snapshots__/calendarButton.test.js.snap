// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test CalendarButton Component 1`] = `
<button
  className="ant-btn"
  onClick={[Function]}
  type="button"
/>
`;
